pkg/apis/camel/v1/build_types.go

```diff
@@ -147,6 +147,7 @@ const (
 // Change format to 'duration' when CRD uses OpenAPI spec v3 (https://github.com/OAI/OpenAPI-Specification/issues/845)
 // +kubebuilder:printcolumn:name="Duration",type=string,JSONPath=`.status.duration`,description="The build last execution duration"
 // +kubebuilder:printcolumn:name="Attempts",type=integer,JSONPath=`.status.failure.recovery.attempt`,description="The number of execution attempts"
+// +kubebuilder:resource:path=builds,scope=Namespaced,shortName=ikb,categories=kamel;camel
```

Comment:
Sorry I may not have be enough specific. The `kubebuilder:resource` marker already exists few lines above (142). Only the `categories` should be appended to the existing marker, instead of duplicating it.

Reply:
Ture, I have now removed the duplication, thanks.